sockeye/rnn.py

```diff
@@ -88,8 +137,13 @@ def get_stacked_rnn(config: RNNConfig, prefix: str) -> mx.rnn.SequentialRNNCell:
                                           dropout_states=config.dropout)
 
         # layer_idx is 0 based, whereas first_residual_layer is 1-based
+        assert not (parallel_inputs and config.residual and config.first_residual_layer > 2), \
```

Comment:
I put is as an assert because it is now a problem with the code. I hope to fix it in the next update.